### PR TITLE
merged sync and splice, removed frob

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/lib/tags.py [hunk 1]
```diff
@@ -43,6 +43,8 @@ def getTufoSubs(tufo, tag):
     prop = '*|%s|%s' % (form,tag)
     if tufo[1].get(prop):
         props.append(prop)
+    else:
+        return props
 
     pref = '*|%s|%s.' % (form,tag)
```
Comment (vEpiphyte) [inline synapse/lib/tags.py]: ```pref = prop + '.'``` is quicker than doing the string % creation a second time.  See https://gist.github.com/vEpiphyte/500ac7ec1ecc8fb019f55c4e7fb82561

File: synapse/reactor.py [hunk 2]
```diff
@@ -46,8 +46,11 @@ def react(self, mesg):
 
             * Handler exceptions *will* propigate upward
```
Comment (vEpiphyte) [inline synapse/reactor.py]: spelling

File: synapse/tests/test_datamodel.py [hunk 3]
```diff
@@ -328,11 +317,21 @@ def test_datamodel_filebase(self):
...
             self.assertRaises(s_datamodel.BadTypeValu, model.getTypeParse, prop, bad)
-            self.assertEqual((None, {}), model.getTypeFrob(prop, bad))
+
+    def test_datamodel_formbase(self):
+
+        modl = s_datamodel.DataModel()
+        modl.addTufoForm('foo:bar')
+        modl.addTufoProp('foo:bar','baz')
+
+        form,base = modl.getPropFormBase('foo:bar:baz')
```
Comment (vEpiphyte) [inline synapse/tests/test_datamodel.py]: should also try a non-existent prop and catch the NoSuchProp error.

File: synapse/cores/common.py [hunk 4]
```diff
@@ -968,32 +713,32 @@ def coresync(self, mesg):
...
+            fd.write(msgenpack(mesg))
+        self.on('splice', save)
 
-        self.on('core:sync', saveobj)
-
-    def eatSyncFd(self, fd):
+    def eatSpliceFd(self, fd):
         '''
-        Consume all cortex sync events from the given file-like object.
+        Consume all cortex splice events from the given file-like object.
```
Comment (invisig0th) [inline synapse/cores/common.py]: Good call.  Will do.

File: synapse/cores/common.py [hunk 5]
```diff
@@ -1036,7 +781,7 @@ def setSaveFd(self, fd, load=True, fini=False):
             core.setSaveFd(fd)
 
         NOTE: This save file is allowed to be storage layer specific.
-              If you want to store cortex sync events, use addSyncFd().
+              If you want to store cortex splice events, use addSpliceFd().
```
Comment (invisig0th) [inline synapse/cores/common.py]: totally

File: synapse/datamodel.py [hunk 6]
```diff
@@ -224,6 +212,25 @@ def addTufoProp(self, form, prop, **info):
...
+        '''
+        Return a form,base tuple for the name parts of a given property.
+
+        Example:
+
+        Args:
+            prop (str): The fully qualified property name
+
+        Returns:
+            (str,str):  The (form,base) name tuple for the prop.
```
Comment (invisig0th) [inline synapse/datamodel.py]: haha, thanks. still getting used to the convention.

File: synapse/lib/scrape.py [hunk 7]
```diff
@@ -3,7 +3,7 @@
 import synapse.data as s_data
 import synapse.lib.datfile as s_datfile
 
-from synapse.common import *
+import synapse.common as s_common
```
Comment (invisig0th) [inline synapse/lib/scrape.py]: yes, definitely.  it's totally my bad that we have a mixed pattern on this :)

File: synapse/lib/tags.py (hunk 1 above)
Comment (invisig0th) [inline synapse/lib/tags.py]: no doubt.  will tweak.

File: synapse/reactor.py (hunk 2 above)
Comment (invisig0th) [inline synapse/reactor.py]: hahaha, thanks

File: synapse/tests/test_datamodel.py (hunk 3 above)
Comment (invisig0th) [inline synapse/tests/test_datamodel.py]: good call.  will add.

**Pull request merged**